mixer/docker/.gitignore

```diff
@@ -0,0 +1,2 @@
+mixs
+ca-certificates.tgz
```

Comment:
How big is this file, really? Can we just commit it? I don't like an external dependency on debian servers for the build dependency.

Reply:
I am checking it in shortly. 
Still WIP..., but comments welcome.

Reply:
Who's going to maintain this ? They do change from time to time ( like expire, get revoked, more get added ).